<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>540</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>WMIT</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QtGLView" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>24</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpenRecent">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Open Recent</string>
     </property>
     <addaction name="actionOpenRecentFile_0"/>
     <addaction name="actionOpenRecentFile_1"/>
     <addaction name="actionOpenRecentFile_2"/>
     <addaction name="actionOpenRecentFile_3"/>
     <addaction name="actionOpenRecentFile_4"/>
     <addaction name="actionOpenRecentFile_5"/>
     <addaction name="actionOpenRecentFile_6"/>
     <addaction name="actionOpenRecentFile_7"/>
     <addaction name="actionOpenRecentFile_8"/>
     <addaction name="actionOpenRecentFile_9"/>
     <addaction name="separator"/>
     <addaction name="actionClearRecentFiles"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuOpenRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Model</string>
    </property>
    <addaction name="actionSetupTextures"/>
    <addaction name="separator"/>
    <addaction name="actionAppendModel"/>
    <addaction name="separator"/>
    <addaction name="actionTakeScreenshot"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRenderer"/>
    <addaction name="actionAnimate"/>
    <addaction name="actionSetTeamColor"/>
    <addaction name="separator"/>
    <addaction name="actionShowModelCenter"/>
    <addaction name="actionShowNormals"/>
    <addaction name="actionShowAxes"/>
    <addaction name="actionShowGrid"/>
    <addaction name="actionShowLightSource"/>
    <addaction name="actionLink_Light_Source_To_Camera"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuModel"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionUVEditor">
   <property name="text">
    <string>UVEditor</string>
   </property>
   <property name="toolTip">
    <string>UVEditor</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSetupTextures">
   <property name="text">
    <string>Setup Textures...</string>
   </property>
  </action>
  <action name="actionAppendModel">
   <property name="text">
    <string>Append Model...</string>
   </property>
  </action>
  <action name="actionAboutApplication">
   <property name="text">
    <string>About...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionRenderer">
   <property name="text">
    <string>Renderer</string>
   </property>
  </action>
  <action name="actionShowModelCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Model Center</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionShowNormals">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Normals</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionTakeScreenshot">
   <property name="text">
    <string>Take Screenshot...</string>
   </property>
  </action>
  <action name="actionShowAxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Axes</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionShowLightSource">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Light Source</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionClearRecentFiles">
   <property name="text">
    <string>Clear List</string>
   </property>
  </action>
  <action name="actionOpenRecentFile_1"/>
  <action name="actionOpenRecentFile_2"/>
  <action name="actionOpenRecentFile_3"/>
  <action name="actionOpenRecentFile_4"/>
  <action name="actionOpenRecentFile_5"/>
  <action name="actionOpenRecentFile_6"/>
  <action name="actionOpenRecentFile_7"/>
  <action name="actionOpenRecentFile_8"/>
  <action name="actionOpenRecentFile_9"/>
  <action name="actionOpenRecentFile_0">
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSetTeamColor">
   <property name="text">
    <string>Set TeamColor...</string>
   </property>
  </action>
  <action name="actionLink_Light_Source_To_Camera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Link Light Source To Camera</string>
   </property>
   <property name="shortcut">
    <string>Shift+L</string>
   </property>
  </action>
  <action name="actionRun_Animation">
   <property name="text">
    <string>Run Animation</string>
   </property>
  </action>
  <action name="actionAnimate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Run Animation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtGLView</class>
   <extends>QWidget</extends>
   <header>QtGLView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
